lib/Service/Sync/SyncService.php

```diff
@@ -114,6 +125,8 @@ public function syncMailbox(Account $account,
 		);
 
 		$this->mailboxSync->syncStats($account, $mailbox);
+		// disconnect imap client
```

Comment:
```suggestion
```

https://stackoverflow.blog/2021/12/23/best-practices-for-writing-code-comments/#h2-b7593c698a940